src/index.tsx

```diff
@@ -1,9 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import { Profile } from "./components/Profile";
 
 const HomePage = () => {
-  return <div>
-    Home
+  return <div style={{
+    display: 'flex',
+    justifyContent: 'center',
+  }}>
```

Comment:
let's consider using [styled-components](https://styled-components.com/) instead of raw style tags